### PR TITLE
Complete website with navigation

### PR DESCRIPTION
**Pull request by carloshs1994**
In this milestone:

- We modify the Awesome books application to have:

    ✅ A Navigation bar.
    ✅ Three content sections:
       ✅ Books list.
       ✅ Add book form.
       ✅ Contact info.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -9,18 +9,46 @@
...
+  <header>
+    <h2>Awesome books</h2>
+    <nav>
+      <div class="list-tab red">List</div>
+      <div>|</div>
+      <div class="form-tab">Add new</div>
+      <div>|</div>
+      <div class="contact-tab">Contact</div>
+    </nav>
+  </header>
```
Comment (moise10r) [inline index.html]: - [x] Good job on implementing all the requirements. Everything looks perfect :clap: . Except one, you have forgotten to add the time function to display the current time and date under the navigation bar. look to the design :point_down:

**Review by moise10r — CHANGES_REQUESTED** [review]
### **Hi @carloshs1994 , @IbrohimRasulov,**  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

### To highlight  
* No linter errors :heavy_check_mark:
 * You have a descriptive README.file Keep rocking :rocket: 
 
### <FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (carloshs1994) [reply index.html]: # Solved
Captura de Pantalla 2022-01-05 a la(s) 16 51 00

**Review by rukundoeric — APPROVED** [review]
Hi @carloshs1994,

_Excellent job on this project. Keep it up 🎆_

_Congratulations! 🎉  on the splendid job 👍
You can now merge this PR and submit the project completion form 🚀 . I wish you all the very best in your next tasks 🥇_


Cheers and Happy coding!👏👏👏

__Feel free to leave any questions or comments in the PR thread if something is not 100% clear.__

------
>As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**